03-task-manager/starter/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "mongodb": "^6.5.0",
```

Comment:
Did this upgrade end up fixing your connection problem?

Reply:
Hasn't been able to solve the problem with this package. My current Node.js version is v20.11.1. I've also tried replacing manually the version of "mongodb": "^6.5.0" to "mongodb": "8.2.3" , but it still doesn't work :c